rero/ng-core
# editor: fix add fields

* Fixes fields (adding in one click).
* Updates angular/cdk and dependencies to 8.2.
* Adds .tgz to gitignore file.

Co-Authored-by: Alicia Zangger <alicia.zangger@rero.ch>
Co-Authored-by: Johnny Mariéthoz <Johnny.Mariethoz@rero.ch>

## Why are you opening this PR?

To fix point 5 of https://github.com/rero/rero-ils-ui/issues/108
To update angular/cdk
To add *.tgz to .gitignore

## How to test?

Got to the record editor and add fields for the list at the left. You should do it in one click.

## Code review check list
